### DISCUSSION TITLE
Export Codesandbox project files in zip folder?

### DISCUSSION THREAD

**Discussion by alimustafa-poder** [codesandbox/codesandbox-client#7314] [General]
Hi, I can't seem to figure out why I can't download the project from Codesandbox. When I click on my project, I am redirected to this screen. Here I am working on my project but I don't see an option to export or download the project files. I have looked on the internet and most of the articles and videos have a burger menu at the top left corner but I don't have any burger menu as you can see in the picture below. Is it because I am using a cloud template instead of a static one?
![gAKlF](https://user-images.githubusercontent.com/90796617/212458065-b467be92-4984-4fcc-879b-0e2894fc1a22.png)

**Comment by hankei6km** [comment]
@alimustafa-poder 

The Cloud Sandbox UI seems to have changed.
I don't even know how to download files with the current UI.

However, I was able to download the files individually(not a archived `.zip` file) by [connecting with VSCode](https://codesandbox.io/docs/learn/repositories/editors#open-your-branch-in-vs-code).

At the time of writing this, it seems that VSCode is unable to connect to Cloud Sandbox.
(Probably a temporary failure, I was usually able to connect with VSCode)
[CodeSandbox Repositories](https://codesandbox.io/docs/learn/repositories/overview) is still accessible from VSCode.

---
Repository opened in CodeSandbox UI
![Repository opened in CodeSandbox UI](https://user-images.githubusercontent.com/1386689/215303980-1cf8efc0-16f1-4757-a9d0-b75223221a61.png)

Connect to the above repository from VSCode and display "Download..." menu
![Connect to the above repository from VSCode and display "Download..." menu](https://user-images.githubusercontent.com/1386689/215304306-8e45c593-0ed5-4b37-bf44-b61f95626837.png)

**Reply by alimustafa-poder** [reply]
@hankei6km Hi, this is exactly how I downloaded my project before I posted this question here. The SSH connection would keep dropping in between and sometimes the connection wouldn't even start. Luckily I had two folders frotend and backend, I simply downloaded both of them because downloading each file seemed impossible (there were too many files so I had to dowload the node moudles with the project totalling 500+ mb). In the end it took me an unimaginable time and patience to download all of the files because as I mentioned earlier the connection would keep dropping.

Edit: Also I tried pushing the project to github but whenever I pushed the project it would just initiate an empty git project with just readme.

Thank you for posting a detailed workaround.

**Comment by typed-sigterm** [comment]
Now it's supported:

<img width="411" height="447" alt="image" src="https://github.com/user-attachments/assets/26fdcc69-f18d-45b1-9ecf-82e919f2643b" />
